contrib/cirrus/lib.sh

```diff
@@ -449,63 +448,43 @@ systemd_banish() {
     $GOSRC/$PACKER_BASE/systemd_banish.sh
 }
 
-install_buildah_packages() {
-    git clone https://github.com/containers/buildah.git /tmp/buildah
-    if [[ -r "$BUILDAH_PACKAGES_FILEPATH" ]]; then
-        source "$BUILDAH_PACKAGES_FILEPATH"
-        req_env_var UBUNTU_BUILDAH_PACKAGES FEDORA_BUILDAH_PACKAGES OS_RELEASE_ID
-        case "$OS_RELEASE_ID" in
-            fedora)
-                $BIGTO ooe.sh sudo dnf install -y ${FEDORA_BUILDAH_PACKAGES[@]}
-                ;;
-            ubuntu)
-                $LILTO $SUDOAPTGET update
-                $BIGTO $SUDOAPTGET install ${UBUNTU_BUILDAH_PACKAGES[@]}
-                ;;
-            *) bad_os_id_ver ;;
-        esac
-    else
-        warn "Could not find $BUILDAH_PACKAGES_FILEPATH in buildah repository root."
-    fi
-}
-
 _finalize() {
     set +e  # Don't fail at the very end
     if [[ -d "$CUSTOM_CLOUD_CONFIG_DEFAULTS" ]]
     then
         echo "Installing custom cloud-init defaults"
-        sudo cp -v "$CUSTOM_CLOUD_CONFIG_DEFAULTS"/* /etc/cloud/cloud.cfg.d/
+        $SUDO cp -v "$CUSTOM_CLOUD_CONFIG_DEFAULTS"/* /etc/cloud/cloud.cfg.d/
     else
         echo "Could not find any files in $CUSTOM_CLOUD_CONFIG_DEFAULTS"
     fi
     echo "Re-initializing so next boot does 'first-boot' setup again."
     cd /
-    sudo rm -rf /var/lib/cloud/instanc*
-    sudo rm -rf /root/.ssh/*
-    sudo rm -rf /etc/ssh/*key*
-    sudo rm -rf /etc/ssh/moduli
-    sudo rm -rf /home/*
-    sudo rm -rf /tmp/*
-    sudo rm -rf /tmp/.??*
-    sudo sync
-    sudo fstrim -av
+    $SUDO rm -rf /var/lib/cloud/instanc*
+    $SUDO rm -rf /root/.ssh/*
+    $SUDO rm -rf /etc/ssh/*key*
+    $SUDO rm -rf /etc/ssh/moduli
+    $SUDO rm -rf /home/*
+    $SUDO rm -rf /tmp/*
+    $SUDO rm -rf /tmp/.??*
```

Comment:
-f here are ignoring errors.  `--force` is a really bad name for this flag. Is that good enough for these builds?  I get nervous when I see /etc files removed this way.

Reply:
In this case I think it's okay, and we **DO** want to ignore missing file/directory errors.  But if I'm overlooking something obvious to you, please let me know.

 This function runs as the very final step when building a Ubuntu or Fedora VM image, to "reset" them for a new "first" use (on next boot).  Not all these files exist on all distros. and I was hesitant to complicate things with lots of conditionals.  In any case, it's not a function for general use during testing.  Perhaps I should comment that?

My other assumption here is, the new VM images will shortly be put to use, so if there is a problem, it will soon get noticed.

Reply:
Okay, I added comments to these functions.